src/main/java/frc/team2412/robot/commands/index/IndexCommand.java

```diff
@@ -0,0 +1,34 @@
+package frc.team2412.robot.commands.index;
+
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.team2412.robot.subsystem.IndexSubsystem;
+
+public class IndexCommand extends SequentialCommandGroup {
+    private final IndexSubsystem subsystem;
+
+    public IndexCommand(IndexSubsystem subsystem) {
+        this.subsystem = subsystem;
+    }
```

Comment:
Need to add `addRequirements(subsystem)` so two commands using the index subsystem don't run at the same time.